bin/desi_dailyproc

```diff
@@ -172,6 +195,18 @@ def main(args):
     sys.stdout.flush()
 
 
+    ###############################                                                                                                                         
+    ### Define desi_proc params ###                                                                                                                             
+    ############################### 
+    cmd_base = 'desi_proc '
+    cmd_base += '--batch '
+    cmd_base += '--traceshift '
+    if args.scatteredlight:
+        cmd_base += '--scattered-light '
+    if args.cameras.lower() != 'none':
+        cmd_base += '--cameras {} '.format(args.cameras)
```

Comment:
Let's use the default args pattern used elsewhere in desi code:  don't specify the default value, in which case it will be `None` (different than the string `"None"`), and then check with code like:
```python
if args.cameras is not None:
    cmd_base += '--cameras {} '.format(args.cameras) 
```
That way you can also avoid the parsing of the cameras argument on lines 95-110 just to make it back into a string that desi_proc can re-parse.